### PR TITLE
Add "m" keyboard shortcut to get to the create my link page

### PR DESCRIPTION
**Pull request by ludeeus**
## Proposed change

Can help our awesome super power users more effectively create my links

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [x] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/state/quick-bar-mixin.ts [hunk 1]
```diff
@@ -43,6 +48,29 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
...
+          window.open(
+            `https://my.home-assistant.io/create-link/?redirect=${path}`,
+            "_blank"
+          );
+
+          return;
+        }
+      }
+      showToast(this, {
+        message: `No matching my link found for ${targetPath}`,
```
Comment (spacegaier) [inline src/state/quick-bar-mixin.ts:70]: Translation would be great 😄

Comment (ludeeus) [reply src/state/quick-bar-mixin.ts:70]: I'll add that if there is a sign of it being accepted.

File: src/state/quick-bar-mixin.ts [hunk 2]
```diff
@@ -43,6 +48,29 @@ export default <T extends Constructor<HassElement>>(superClass: T) =>
       showQuickBar(this, { commandMode });
     }
 
+    private _createMyLink(e: KeyboardEvent) {
+      if (!this._canOverrideAlphanumericInput(e) || !this.hass) {
+        return;
+      }
+      const targetPath = mainWindow.location.pathname;
+
+      for (const [path, redirect] of Object.entries(
```
Comment (balloob) [inline src/state/quick-bar-mixin.ts:57]: It's a slug, not a path
```suggestion
      for (const [slug, redirect] of Object.entries(
```

File: src/state/quick-bar-mixin.ts (hunk 1 above)
Comment (balloob) [reply src/state/quick-bar-mixin.ts:70]: I like the idea.

File: src/state/quick-bar-mixin.ts [hunk 3]
```diff
@@ -1,11 +1,15 @@
 import type { PropertyValues } from "lit";
 import tinykeys from "tinykeys";
+import { isComponentLoaded } from "../common/config/is_component_loaded";
+import { mainWindow } from "../common/dom/get_main_window";
 import {
   QuickBarParams,
   showQuickBar,
 } from "../dialogs/quick-bar/show-dialog-quick-bar";
+import { getMyRedirects } from "../panels/my/ha-panel-my";
```
Comment (balloob) [inline src/state/quick-bar-mixin.ts:9]: This import should be loaded dynamically.

**Review by balloob — APPROVED** [review]
Nice! 

For a future PR we might want to consider adding this to quick bar too, to improve discoverability of this feature.

**Comment by frenck** [comment]
Sounds like a nice "Tip" too 😁

**Pull request merged**